x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/IndexStatus.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.profiling;
+
+enum IndexStatus {
```

Comment:
This has been moved from an inner enum to top-level. The only change is that we have added the `TOO_OLD` value.